addons/mail/static/tests/tours/mail_full_composer_test_tour.js

```diff
@@ -0,0 +1,53 @@
+odoo.define('mail/static/tests/tours/mail_full_composer_test_tour.js', function (require) {
+"use strict";
+
+const tour = require('web_tour.tour');
+
+/**
+ * This tour depends on data created by python test in charge of launching it.
+ * It is not intended to work when launched from interface. It is needed to test
+ * an action (action manager) which is not possible to test with QUnit.
+ * @see mail/tests/test_mail_full_composer.py
+ */
+tour.register('mail/static/tests/tours/mail_full_composer_test_tour.js', {
```

Comment:
According to me this is mainly noise; adding tests is ok, adding tests for a commented line only slows down runbot and add nothing. Moreover the "Re: blah" is already tested in python unit tests (much quicker, and along with other tests).

Reply:
The full composer feature is considered important enough and it is currently not working at all. There are also related tasks 2291893 and 2291887 that should introduce more fixes to it (maybe to be done directly in current PR too).

It's not acceptable to have a feature like this one to be completely broken and runbot to be green.

Unfortunately introducing a tour (as opposed to writing a QUnit) is the only way we have to test this one due to the current implementation of the web client / action manager.

I hear your concern and I'm indeed aware of the slowness of tours, but I think in this case it's necessary / worth the time on runbot.

Reply:
Well actually this should not have passed review state if something like the composer is "currently nor working at all". I don't understand why a 2 lines fix about commented lines require a test, because there is no chance someone will remove those lines and it is not a corner case. Unless you plan to improve this test with other corner cases with other fixes, in which case it seems ok.

It fear adding too much tests for details may feel like we don't have to manually test anymore or ignore functional flows.

A global tour of composer is a good idea anyway, but I think it should be done in a real functional addon, like a complete flow of sale order confirmation + send by email + suggested recipients + email status .

Reply:
Just to be sure everyone is talking about the same thing : composer is globally working, SEB is talking about the "full size"  composer in modal which was not working and we didn't know about it. (If it was clear for everyone, sorry for that useless comment)

One of our goal (I think) is to have tours to test all mail functionalities as there are some features we can't totally cover with QUnit tests. The thing is that we are currently running out of time (like everybody I think) to add full tours. So this tour is very light for now but could be enriched later.

(@seb-odoo correct me if I'm wrong here)

Reply:
Yes I think this is pretty much the idea. I would find great if we had a real tour with a complete flow such as the one of sales order as suggested, but we have to start somewhere, and doing directly the full tour is out of scope of current PR.

@xdu-odoo you can have a look at the other tasks I referenced above, maybe those 2 are worth immediately doing as part of this PR / this tour.